### ISSUE TITLE
Project config on GCP file

### ISSUE THREAD

**Issue by adawolfs** [heptio/ark#400]
I have found that there is a commit to remove the project config from ark config file and get it from credentials but its actually not supported on examples, and giving this error

> An error occurred: rpc error: code = Unknown desc = missing project in gcp configuration

Please use this template
```
apiVersion: ark.heptio.com/v1
kind: Config
metadata:
  namespace: heptio-ark
  name: default
persistentVolumeProvider:
  name: gcp
  config:
    project: <YOUR_PROJECT> #Replace by your project id
backupStorageProvider:
  name: gcp
  bucket: <YOUR_BUCKET> #Replace by your bucket name
backupSyncPeriod: 30m
gcSyncPeriod: 30m
scheduleSyncPeriod: 1m
restoreOnlyMode: false
```

**Comment by j0npulsifer** [comment]
Ran into this issue tonight.. solved it by using a newer image from [`gcr.io/heptio-images/ark`](https://console.cloud.google.com/gcr/images/heptio-images/GLOBAL/ark?gcrImageListsize=5)

**Comment by ncdc** [comment]
Hi @adawolfs and @j0npulsifer, please make sure you are using the examples and docs from v0.7.1: https://github.com/heptio/ark/tree/v0.7.1. That version still requires the gcp project in the config.

If you instead want to use a copy that no longer requires the gcp project, you'll need to use the `master` image tag instead of `latest`.

**Comment by skriss** [comment]
FYI: we just released v0.8.0 which does not require `project` to be specified. Closing this issue as resolved, but feel free to reach out if there are any further issues.
